src/main/java/org/myrobotlab/service/Solr.java

```diff
@@ -463,6 +467,34 @@ public IplImage bytesToImage(byte[] bytes) throws IOException {
     // Scala
   }
 
+  /**
+   * deserialize from a png byte array to a base64 encoded string
+   * for display inline in html.
+   * 
+   * @param bytes
+   *          input bytes
+   * @return a string
+   * @throws IOException
+   *           boom
+   * 
+   */
+  public String bytesToBase64Jpg(byte[] bytes) {
+    //
+    // let's assume we're a buffered image .. those are serializable :)
+    ByteArrayOutputStream os = new ByteArrayOutputStream();
+    try {
+      BufferedImage bufImage = ImageIO.read(new ByteArrayInputStream(bytes));
+      ImageIO.write(bufImage, "jpg", os);

```

Comment:
I swore I made this in CodecUtils at one point 
It'd be "nice" if it was re-implemented there
```java
public String toBase64Image(byte[] bytes){
         toBase64Image(bytes, "jpg")
}
public String toBase64Image(byte[] bytes, String type)
```
if this was done again in CodecUtils we'd standardize it, and this way anyone can use it for gif tiff png and anything else ImageIO currently supports